### DISCUSSION TITLE
Vercel Banned ME :)

### DISCUSSION THREAD

**Discussion by rinrinx2** [spencerwooo/onedrive-vercel-index#459] [Q&A] [answered]
![Screenshot_2](https://user-images.githubusercontent.com/58888200/154422998-97ce4a69-e43b-41d4-b781-86fadcccaa40.png)

**Comment by spencerwooo** [comment, marked as answer]
Check your other projects, I highly doubt that it's _this_ project that caused Vercel to ban you. Or were you serving DMCA or copyright protected files? In either case I cannot help you.
